dotcom-rendering/src/web/server/articleToHtml.tsx

```diff
@@ -207,6 +208,13 @@ export const articleToHtml = ({ data }: Props): string => {
 		}(document, "script", "twitter-wjs"));
 	`;
 
+	const url = CAPIArticle.webURL;
+
+	const recipeMarkup =
+		url in recipeSchema
+			? `<script type="application/ld+json">${recipeSchema[url]}</script>`
```

Comment:
Could this live in `articleTemplate`? There's a number of examples of similar code being done inline